torchtitan/experiments/flux/train.py

```diff
@@ -204,6 +209,46 @@ def train_step(self, input_dict: dict[str, torch.Tensor], labels: torch.Tensor):
 
         self.metrics_processor.log(self.step, global_avg_loss, global_max_loss)
 
+        # Evaluate the model during training
+        if (
+            self.step % self.job_config.eval.eval_freq == 0
+            or self.step == self.job_config.training.steps
+        ):
+            self._eval_flux_model()
```

Comment:
I'd suggest we do
```suggestion
            model.eval()
            self._eval_flux_model()
            model.train()
```
instead of doing it in couple of different places